src/LoginPage/components/RegisterForm.js

```diff
@@ -21,36 +22,33 @@ import "../css/register.css";
 import createFragment from "react-addons-create-fragment";
 import { ValidatorComponent } from 'react-material-ui-form-validator';
 import holder from "../images/holder.jpg";
+import { fetchMeetingOptions } from "../../actions";
 
+function mapStateToProps ({ allMeetingOptions }){
+    return {allMeetingOptions}
+}
 class RegisterForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
       user: {},
       imgURL:
-        "https://image.freepik.com/free-icon/female-student-silhouette_318-62252.jpg"
+        "https://image.freepik.com/free-icon/female-student-silhouette_318-62252.jpg",
```

Comment:
Is this a good candidate to either place within the project as an asset, or to use a configuration setting to link to the image. I'd recommend not linking directly to a resource which may change.

Reply:
Yeah, I would suggest the same thing. We shouldn't be requesting images from the internet. We can add it to the public directory and reference it from there.